en/07-prompt-guardrails/scripts/public-repo-check.sh

```
@@ -15,10 +15,18 @@
 # either one. For blocking at the source (never letting it be read in the
 # first place), see settings.json's permissions.deny.
+#
+# Found 2026-09-01: this script is verify.yml's only build-blocking check,
+# but there was no way to mark "reviewed and deliberately left in" (e.g.
+# synthetic test data using an RFC 2606 reserved domain) — so a single such
+# value made it exit 1 forever. public-repo-check-allowlist.txt makes that
+# distinction explicit. Any match not on the allowlist still blocks the
+# build as-is.
 
 set -uo pipefail
 cd "$(git rev-parse --show-toplevel)"
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)